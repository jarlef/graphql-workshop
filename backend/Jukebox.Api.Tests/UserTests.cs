using Jukebox.Api.Tests.Infrastructure;
using Snapshooter.Xunit;

namespace Jukebox.Api.Tests;

public class UserTests(JukeboxApplicationFactory factory) : BaseIntegrationTest(factory)
{
    [Fact]
    public async Task GetUser_ShouldReturnUser()
    {
        var result = await Client.GetUser.ExecuteAsync();
        result.Data.MatchSnapshot();
    }
}
