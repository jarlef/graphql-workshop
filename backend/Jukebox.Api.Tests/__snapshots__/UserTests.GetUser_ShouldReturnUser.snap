{
  "Me": {
    "Id": 1,
    "Name": "Mr. Jukebox"
  }
}
